mono/mini/mini-llvm.c

```diff
@@ -7508,6 +7582,15 @@ emit_method_inner (EmitContext *ctx)
 
 	mono_llvm_add_func_attr (method, LLVM_ATTR_UW_TABLE);
 
+	if (!cfg->llvm_only) {
+		// FIXME/Refine: 
+		//  I was seeing a lot of weird behavior in the corlib tests that suggested a few edge cases to consider when enabling LLVM inlining.
+		//  I think that enabling inlining for the majority of methods is a good idea.
+		//  In the past, we weren’t inlining any methods because we were obfuscating all of the call sites. This preserves that behavior.
+		//  Also not sure how letting LLVM do inlining impacts things like non-C++ EH. There are a number of correctness hurdles there.
+		mono_llvm_add_func_attr (method, LLVM_ATTR_NO_INLINE);
```

Comment:
Is this still needed ? I think we were doing inlining, but only for direct calls.

Reply:
This prevents LLVM from automatically inlining the direct call. I can revisit the issue, but I expected it to be a separate set of fixes from these. We didn't have direct calls before, which prevented any LLVM-side inlining. So this should preserve that behavior for now.